context/app/static/js/components/detailPage/ProcessedData/ProcessedDataset/DatasetTitle.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Typography from '@mui/material/Typography';
+import { useHandleCopyClick } from 'js/hooks/useCopyText';
+import { SecondaryBackgroundTooltip } from 'js/shared-styles/tooltips';
+import IconButton from '@mui/material/IconButton';
+import ContentCopyIcon from '@mui/icons-material/ContentCopyRounded';
+import Stack from '@mui/material/Stack';
+import StatusIcon from '../../StatusIcon';
+import { useProcessedDatasetContext } from './ProcessedDatasetContext';
+import VersionSelect from '../../VersionSelect';
+import { useTrackEntityPageEvent } from '../../useTrackEntityPageEvent';
+import SummaryJSONButton from '../../summary/SummaryJSONButton';
+
+export function DatasetTitle() {
+  const {
+    dataset: { hubmap_id, status, uuid, entity_type },
+  } = useProcessedDatasetContext();
+  const copyText = useHandleCopyClick();
+  const track = useTrackEntityPageEvent();
+  return (
+    <Typography variant="h5" display="flex" alignItems="center" gap={0.5}>
+      <StatusIcon status={status} />
+      {hubmap_id}
+      <SecondaryBackgroundTooltip title="Copy HuBMAP ID">
+        <IconButton
+          onClick={() => {
```

Comment:
Should we also use `useCallback` here? I'm fine with either, but we should be consistent.